src/modules/simulation/gz_bridge/GZMixingInterfaceServo.cpp

```diff
@@ -33,6 +33,9 @@
 
 #include "GZMixingInterfaceServo.hpp"
 
+#define SERVO_OUTPUT_SCALING (10.)
+#define SERVO_OUTPUT_OFFSET (180.)
```

Comment:
Why is it necessary to have this as an ifdef?

Reply:
Should be `static constexpr`

Reply:
Updated it to static constexpr, thanks!